docs/kubernetes/user-guides/snapshots.md

```diff
@@ -1,8 +1,8 @@
-# Kubernetes Snapshots User Guide (Alpha)
+# Kubernetes Snapshots User Guide (Beta)
 
->**Attention:** VolumeSnapshot is an alpha feature. Make sure you have enabled it in Kubernetes API server using `--feature-gates=VolumeSnapshotDataSource=true` flag.
+>**Attention:** Attention: VolumeSnapshot is a Beta feature enabled by default in Kubernetes 1.17+. Attention: VolumeSnapshot is only available in the driver version "master".
 
-### Install Driver with alpha snapshot feature
+### Install Driver with beta snapshot feature
 
 1. [One-time per project] Create GCP service account for the CSI driver and set required roles
```

Comment:
We can link to the install driver user guide for this step

Reply:
Ack